src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.id);
+    default:
```

Comment:
- [ ] Well done implementing the redux actions, action creators, reducers and combining them in your store.js. I think it would be better if you do the same for the categories.js and import the categoriesReducer to combine with bookReducer as you can also see in the project requirements below 👇 
![Screenshot (241)](https://user-images.githubusercontent.com/67087664/160648964-c9be760a-9fff-422b-a743-b055589c9d00.png)

Reply:
Oh, yes. Forgot that. Will do so, thank you.

Reply:
This issue is well resolved! ✔️